bin/fetch_and_pickle_schedule.py

```diff
@@ -7,16 +7,16 @@
 import os
 import pickle
 import sys
-from datetime import datetime
+from datetime import date, timedelta
+import dateutil.parser
+import pytz
 
-import pendulum
 import requests
```

Comment:
The common python format is:
```
stdlib imports
<newline>
thirdparty imports
<newline>
local imports
```

So both dateutil and pytz should be together with requests (in sorted order), not with stdlib.

Reply:
Done.